A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -1103,6 +1138,67 @@ var A320_Neo_UpperECAM;
                             }
                         ]
 
+                    },
+                    {
+                        name: "Fuel",
+                        messages: [
+                            {
+                                message: "R WING TK LO LVL",
+                                level: 2,
+                                flightPhasesInhib: [3, 4, 5, 7, 8, 9],
+                                isActive: () => {
+                                    const quantityL = SimVar.GetSimVarValue("FUEL TANK LEFT MAIN QUANTITY", "gallons") * this.gallonToKg;
+                                    const quantityR = SimVar.GetSimVarValue("FUEL TANK RIGHT MAIN QUANTITY", "gallons") * this.gallonToKg;
+                                    return (quantityR < 750) && (quantityL > 750);
+                                },
+                                actions: this.getLoWingTankActions("R")
+                            },
+                            {
+                                message: "L WING TK LO LVL",
+                                level: 2,
+                                flightPhasesInhib: [3, 4, 5, 7, 8, 9],
+                                isActive: () => {
+                                    const quantityL = SimVar.GetSimVarValue("FUEL TANK LEFT MAIN QUANTITY", "gallons") * this.gallonToKg;
+                                    const quantityR = SimVar.GetSimVarValue("FUEL TANK RIGHT MAIN QUANTITY", "gallons") * this.gallonToKg;
+                                    return (quantityL < 750) && (quantityR > 750);
+                                },
+                                actions: this.getLoWingTankActions("L")
+                            },
+                            {
+                                message: "CTR TK PUMPS OFF",
+                                level: 2,
+                                flightPhasesInhib: [1, 3, 4, 5, 7, 8, 9, 10],
+                                isActive: () => (!SimVar.GetSimVarValue("FUELSYSTEM PUMP SWITCH:1", "boolean") || !SimVar.GetSimVarValue("FUELSYSTEM PUMP SWITCH:4", "boolean")) && !SimVar.GetSimVarValue("L:A32NX_ECAM_INOP_SYS_FUEL PUMPS", "Bool") && SimVar.GetSimVarValue("FUEL TANK CENTER QUANTITY", "gallons") * this.gallonToKg > 250,
```

Comment:
Maybe break this into multiple lines, it's veeery long.

Reply:
changed. :)